BlueprintUI/Sources/Element/ElementState.swift

```diff
@@ -622,6 +634,9 @@ final class ElementState {
             $0.wasVisited = false
             $0.hasUpdatedChildrenDuringLayout = false
             $0.clearOrderedChildren()
+            if !Self.enableCrossLayoutCaching {
```

Comment:
Should we do this at the _end_ of the layout, so we clear out data and reduce memory usage?

Reply:
Done